services/drupal/composer.lock

```diff
@@ -1675,16 +1675,16 @@
         },
         {
             "name": "doctrine/collections",
-            "version": "1.7.3",
+            "version": "1.8.0",
```

Comment:
I'm very suspicious of these unrelated dependency updates.  I wonder how they crept in here.  Just running `composer require drupal/content_moderation_notifications:dev-3.x` doesn't lead to these dependencies changing. Any thoughts on what might be happening here?  This comment is relevant for this package as well as several others like phpdocumentor/type-resolver, doctrine/event-manager, and others.